rpc/rpcserver/server.go

```diff
@@ -294,6 +294,7 @@ func (s *walletServer) Balance(ctx context.Context, req *pb.BalanceRequest) (
 
 	account := req.AccountNumber
 	reqConfs := req.RequiredConfirmations
+
```

Comment:
Can you remove this extra newline? no reason to touch this file now from this pr.